lambdas/validator.py

```diff
@@ -1,32 +1,19 @@
 import json
 import logging
-import os
 
-import sentry_sdk
-from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+from lambdas.config import Config, check_verbosity, configure_logger
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
-env = os.getenv("WORKSPACE")
-if sentry_dsn := os.getenv("SENTRY_DSN"):
-    sentry = sentry_sdk.init(
-        dsn=sentry_dsn,
-        environment=env,
-        integrations=[
-            AwsLambdaIntegration(),
-        ],
-        traces_sample_rate=1.0,
-    )
-    logger.info("Sentry DSN found, exceptions will be sent to Sentry with env=%s", env)
-else:
-    logger.info("No Sentry DSN found, exceptions will not be sent to Sentry")
+CONFIG = Config()
 
 
 def lambda_handler(event: dict) -> str:
-    if not os.getenv("WORKSPACE"):
-        unset_workspace_error_message = "Required env variable WORKSPACE is not set"
-        raise RuntimeError(unset_workspace_error_message)
+    """AWS Lambda entrypoint."""
+    CONFIG.check_required_env_vars()
+    verbose = check_verbosity(event.get("verbose", False))
```

Comment:
Hmm, small and simple function, but I'm wondering if we can think of a different name than "check_verbosity"? 🤔 Maybe..."get_verbose_level" or "parse_log_level"?

Reply:
Opting to keep this, as it was cribbed from other lambda projects.  And given that it's fairly tightly coupled with the `{..."verbose":true|false...}` potential in a lambda input payload, I feel like it kind of works?

But I also agree. To me, this was one of the more awkward areas of the lambda templates and previous implementations.  I hope it's okay to move forward with it as-is, with the acknowledgement it's not ideal.